src/components/SlateEditor/plugins/concept/SearchConceptResults.tsx

```diff
@@ -72,35 +57,57 @@ interface Props {
+  }
   return (
-    <div>
-      {searching && <Spinner appearance="absolute" />}
-      {results.length === 0 ? <p>{t(`searchPage.conceptNoHits`, { query: searchObject.query })}</p> : null}
+    <StyledList>
       {results.map((result) => (
-        <StyledConceptResult key={result.id}>
-          {result.glossData ? <StyledGlobe /> : <StyledConcept />}
-          <StyledConceptResultHeader>
-            {(result.glossData
-              ? `${t(`languages.${result.glossData.originalLanguage}`)}: ${result.glossData.gloss}`
-              : result.title.title) ?? t("conceptSearch.noTitle")}
-            {(result.status.current === "PUBLISHED" || result.status.other.includes("PUBLISHED")) && (
-              <StyledCheckIcon aria-label={t("form.workflow.published")} title={t("form.workflow.published")} />
-            )}
-          </StyledConceptResultHeader>
-          <StyledConceptContent>
-            {result.glossData ? result.title.title : result.content.content ?? t("conceptSearch.noContent")}
-          </StyledConceptContent>
-          <StyledButton
-            onClick={(evt) => {
-              evt.stopPropagation();
-              addConcept(result);
-            }}
-          >
-            {t(`form.content.${result.conceptType}.choose`)}
-          </StyledButton>
-        </StyledConceptResult>
+        <ListItemRoot key={result.id} context="list" nonInteractive asChild consumeCss>
+          <li>
+            <StyledListItemImage src="" alt="" fallbackElement={result.glossData ? <Globe /> : <Concept />} />
+            <StyledListItemContent>
+              <ListItemContent>
+                <ListItemHeading>
+                  {(result.glossData
+                    ? `${t(`languages.${result.glossData.originalLanguage}`)}: ${result.glossData.gloss}`
+                    : result.title.title) ?? t("conceptSearch.noTitle")}
+                </ListItemHeading>
+                {(result.status?.current === PUBLISHED || result.status?.other.includes(PUBLISHED)) && (
+                  <SafeLinkIconButton
+                    size="small"
+                    variant="success"
+                    target="_blank"
+                    aria-label={t("form.workflow.published")}
+                    title={t("form.workflow.published")}
+                    to={`${config.ndlaFrontendDomain}/concept/${result.id}`}
+                  >
+                    <CheckLine />
+                  </SafeLinkIconButton>
+                )}
+              </ListItemContent>
+              <ListItemContent>
+                {!!result.content.content.length && (
+                  <StyledText textStyle="body.small">{result.content.content}</StyledText>
+                )}
+                <Button
+                  onClick={(e) => {
+                    e.stopPropagation();
+                    addConcept(result);
+                  }}
+                >
+                  {t("form.choose")}
+                </Button>
```

Comment:
Kunne denne blitt small? syntes den er veldig høy :/ og skulle vi prøve å legge de to knappene ved siden av hverandre og nederst? Så blir det tilsvarende det Hedvig tegnet opp til kortene i struktur?